lang/pl/content.php

```diff
@@ -173,4 +174,53 @@
             "title_1" => "Szukamy właśnie Ciebie?",
         ],
     ],
+    "contact" => [
+        "title_1" => "Napisz",
+        "title_2" => "do Nas!",
```

Comment:
```suggestion
        "title_2" => "do nas!",
```

Reply:
In Polish pronouns like "ja, my, nasz, mój" we write lowercase.

Reply:
Fixed.